### DISCUSSION TITLE
git clone issue !!

### DISCUSSION THREAD

**Discussion by jonluca7** [community/community#113592] [Programming Help]
### Select Topic Area

General

### Body

Hi , 

please i need help . i'm training as Android Mobile App Developer . 

I got an issue when using git clone . i received this following message :

<< fatal: unable to access 'https://git.example.com/example.git (you can omit this URL)/': URL rejected: Malformed input to a URL function >>

i don't know how to solve this issue . Hope to receive some help.

Thanks 

Regard

**Comment by brkykb** [comment]


**Reply by queenofcorgis** [reply]
@brkykb, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by waresnew** [comment]
It looks like you accidentally included `(you can omit this URL)` in the link to be cloned. `git clone` downloads a repository from a link/URL, so the error was basically saying you supplied an invalid link, because you added `(you can omit this URL)` at the end.

I think you ran `git clone 'https://git.example.com/example.git (you can omit this URL)'` instead of `git clone https://git.example.com/example.git`. Running the latter command (replacing the example link with your own) should work.
